app/controllers/validators_controller.rb

```diff
@@ -115,6 +115,8 @@ def set_validator
       network: params[:network],
       account: params[:account]
     ).first or not_found
+  rescue ActiveRecord::RecordNotFound
```

Comment:
Same as above

Reply:
One additional question here - Can we do that ? 
```suggestion
 @validator = Validator.where(
      network: params[:network],
      account: params[:account]
    ).first or render_404
```